### PR TITLE
Sort a stack -documentation added

### LINKED ISSUE #7164: 4.1.9.1-Stack: Sort a stack -documentation
## Description
write an algorithm to sort a stack

## Task
- [ ] Documentation
- [ ] Audio
- [ ] Video

## Note
- While requesting the issue to be assigned, please mention your **NAME & BATCH NUMBER**.
- Kindly refer to [Issue Guidelines](https://github.com/girlscript/winter-of-contributing/blob/DSA/DSA/CONTRIBUTING.md#issues).
- In case you see that the issue has already been assigned, try to request for a different task within the same issue.
- Changes should be made inside the ````DSA/directory```` & ````DSA```` branch.
- Name of the file should be same as the issue.
- This issue is only for 'GWOC' contributors of 'DSA' Domain.

### Domain

DSA

### Type of Contribution

 Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by uzma024**
<hr>

## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
Fixes #7164

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [x] DSA
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

### CODE REVIEW

**Comment by uzma024** [comment]
@Muskan-creator @nimisha-yadav Kindly review and merge my pr.

**Comment by Muskan-creator** [comment]
Hey @uzma024 .The issue was for documentation  not video contribution

File: DSA/4.1 Stacks/01. Introduction/3.Introduction_to_Stacks_video.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## Introduction to stacks:
```
Comment (Muskan-creator) [inline DSA/4.1 Stacks/01. Introduction/3.Introduction_to_Stacks_video.md:1]: Kindly remove this file of stack and remain that file only which was for sorting a stack

File: DSA/4.1 Stacks/01. Introduction/3.Introduction_to_Stacks_video.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+ A stack is an Abstract Data Type (ADT), commonly used in most programming languages. It is named stack as it behaves like a real-world stack, for example – a deck of cards or a pile of plates, etc.
+
+ ## Link for the Video(click on the image):
+
+ [![image](https://i.ibb.co/6mC1LRX/Screenshot-2021-11-14-at-11-48-47-PM.png)](https://youtu.be/nwNHV98-nao)
```
Comment (Muskan-creator) [inline DSA/4.1 Stacks/01. Introduction/3.Introduction_to_Stacks_video.md:6]: There is no need to add a video on stack that issue was already assigned or created

**Review by Muskan-creator — CHANGES_REQUESTED** [review]

File: DSA/4.1 Stacks/09. Sort a stack/1.Sort_a_stack _documentation.md [hunk 3]
```diff
@@ -0,0 +1,271 @@
+# <u>Sorting A Stack</u>
+
+## Stack:
+    Stack is a data structure that follows Last In First Out(LIFO) methord for
```
Comment (Muskan-creator) [inline DSA/4.1 Stacks/09. Sort a stack/1.Sort_a_stack _documentation.md:4]: Spelling mistake in method

**Comment by uzma024** [comment]
I am not able to create changes here and its a bit too messy. So I have created another pull request.
Kindly review it. 
https://github.com/girlscript/winter-of-contributing/pull/7959#issue-1054875856

**Pull request closed without merging**